phpunit/navigation-block-fallback-test.php

```diff
@@ -0,0 +1,151 @@
+<?php
+/**
+ * Tests Fallback Behavior for Navigation block
+ *
+ * @package WordPress
+ */
+
+class Tests_Block_Navigation_Fallbacks extends WP_UnitTestCase {
+
+	public function set_up() {
+		parent::set_up();
+		switch_theme( 'emptytheme' );
+	}
+
+	public function get_navigations_in_database() {
+		$navs_in_db = new WP_Query(
+			array(
+				'post_type'      => 'wp_navigation',
+				'post_status'    => 'publish',
+				'posts_per_page' => -1,
+				'orderby'        => 'date',
+				'order'          => 'DESC',
+			)
+		);
+
+		return $navs_in_db->posts ?? array();
+	}
+
+	public function test_gets_fallback_navigation_with_existing_navigation_menu_if_found() {
+
+		$navigation_post_1 = self::factory()->post->create_and_get(
+			array(
+				'post_type'    => 'wp_navigation',
+				'post_title'   => 'Existing Navigation Menu 1',
+				'post_content' => '<!-- wp:page-list /-->',
+			)
+		);
+
+		$navigation_post_2 = self::factory()->post->create_and_get(
+			array(
+				'post_type'    => 'wp_navigation',
+				'post_title'   => 'Existing Navigation Menu 2',
+				'post_content' => '<!-- wp:page-list /-->',
+			)
+		);
+
+		$most_recently_published_nav = $navigation_post_2;
+
+		$fallback = gutenberg_block_core_navigation_create_fallback();
+
+		$this->assertEquals( $fallback->post_title, $most_recently_published_nav->post_title );
+		$this->assertEquals( $fallback->post_type, $most_recently_published_nav->post_type );
+		$this->assertEquals( $fallback->post_content, $most_recently_published_nav->post_content );
```

Comment:
This is also the same in both posts, so the test seems a bit pointless!

Reply:
haha yes!